brianlong/www.validators.app
# 181679720_move_ftx_and_coin_gecko_clients_to_proper_folder

#### What's this PR do?
- Move ftx and coin gecko clients to api_clients namespace

#### How should this be manually tested?
- Run tests
- Run on staging, wait one night, check if the new prices were gathered

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/181679720

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
